beacon-chain/core/blocks/payload_test.go

```diff
@@ -409,6 +413,106 @@ func Test_ExecutionEnabled(t *testing.T) {
 	}
 }
 
+func Test_ValidatePayloadWhenMergeCompletes(t *testing.T) {
+	tests := []struct {
+		name    string
+		payload *ethpb.ExecutionPayload
+		header  *ethpb.ExecutionPayloadHeader
+		err     error
+	}{
+		{
+			name:    "merge incomplete",
+			payload: emptyPayload(),
+			header:  emptyPayloadHeader(),
+			err:     nil,
+		},
```

Comment:
How does this setup ensure that we are pre-merge?

Reply:
https://github.com/prysmaticlabs/prysm/pull/10028/files#diff-abcfccaaeba0eafc1bfaf19006a05c7591ec4711e053781094f1e68b7aa9ebaaR28